backend/src/interfaces.ts

```diff
@@ -22,15 +24,14 @@ export interface Game {
 }
 
 export interface User {
-  id: string;
   username: string;
   password: string;
   salt: string;
   profilePicture?: string;
   highScore?: number;
   cumulativeScore?: number;
   shirts?: number; // Would this refer to the number of secrets(clothes drawing) found?
-  dateJoined?: Date;
+  dateJoined: Date | Timestamp;
```

Comment:
probably should be one or the other

Reply:
ok so, firebase converts the Date to a Timestamp so I have that so I can convert it back when I return the user info without it complaining